runfolder/services.py

```diff
@@ -153,13 +162,30 @@ def get_runfolder_state(self, runfolder):
         If the file .arteria/state exists, it will determine the state. If it doesn't
         exist, the existence of the marker file RTAComplete.txt determines the state.
         """
-        completed_marker_file = self._configuration_svc["completed_marker_file"]
+        completed_marker_files = None
+        try:
+            completed_marker_files = self._configuration_svc["completed_marker_file"]
+            if isinstance(completed_marker_files, basestring):
```

Comment:
BTW, do we have a policy regarding Python2/3? Because `basestring` doesn't exist in Python3 anymore, instead one would use `str` here.
The reason I use `basestring` is because it is a parent class to both `str` and `unicode`.
@johandahlberg

Reply:
Right now we run this with Python 2, but I do think that we should see about upgrading all the Arteria services to Python 3. I think that you will be fine with using `str` in this particular case however.

Reply:
alright, I replaced it with `str`